### PR TITLE
Network tc

### PR DESCRIPTION
**Pull request by deepaksrma**

### CODE REVIEW

File: apps/examples/testcase/le_tc/network/tc_net_poll.c [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+#include <stdio.h>
+#include <netinet/in.h>
+#include <netutils/netlib.h>
+#include <sys/socket.h>
+#include <lwip/sockets.h>
+#include <tinyara/os_api_test_drv.h>
+#include <sys/ioctl.h>
+#include <poll.h>
+#include "tc_internal.h"
```
Comment (thapav) [inline apps/examples/testcase/le_tc/network/tc_net_poll.c:32]: Please follow below header format:

##HEADEER FORMAT
<tinyara/config.>
standard headers
TizenRT(TinyAra) specific headers
framework or external headers
Local headers
Please place the header file like below sequence.
************************
tinyara/config.h
standard header : stdio.h, unistd.h, string.h and so on.
tinyara/xxx.h
absolute path header : "xxx.h"
************************

File: apps/examples/testcase/le_tc/network/tc_net_poll.c [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+	struct poll_test_args args;
+
+	sd = socket(AF_INET, SOCK_DGRAM, 0);
+
+	args.fd = sd;
+	args.fds = NULL;
+	args.setup = 1;
+
+	ret = ioctl(fd, TESTIOC_NET_POLL, &args);
+	TC_ASSERT_LEQ("lwip_poll", ret, -1);
```
Comment (thapav) [inline apps/examples/testcase/le_tc/network/tc_net_poll.c]: Do you think following this is better ?
 TC_ASSERT_LEQ_CLEANUP("lwip_poll", ret, -1, close(sd));

File: apps/examples/testcase/le_tc/network/tc_net_poll.c [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+	fds[0].fd = sd;
+	args.fds = 0;
+
+	args.fd = sd;
+	args.fds = &fds;
+	args.setup = 1;
+
+	ret = ioctl(fd, TESTIOC_NET_POLL, &args);
+	TC_ASSERT_GEQ("lwip_poll", ret, 0);
```
Comment (thapav) [inline apps/examples/testcase/le_tc/network/tc_net_poll.c]: same as above

File: os/drivers/os_api_test/network/test_net_poll.c [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
```
Comment (thapav) [inline os/drivers/os_api_test/network/test_net_poll.c]: 2021*

File: os/drivers/os_api_test/network/test_net_poll.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <debug.h>
+#include <tinyara/os_api_test_drv.h>
+#include <lwip/sockets.h>
+#include <poll.h>
+#include <sys/types.h>
```
Comment (thapav) [inline os/drivers/os_api_test/network/test_net_poll.c]: Same, please check header format

File: os/drivers/os_api_test/network/test_net_poll.c [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+static int test_poll(unsigned long arg)
+{
+	int ret = -1;
+	poll_test_args_t *args = (poll_test_args_t *)arg;
+
+	ret = lwip_poll(args->fd, args->fds, args->setup);
+	return ret;
+}
+
```
Comment (thapav) [inline os/drivers/os_api_test/network/test_net_poll.c]: Add function description

File: apps/examples/testcase/le_tc/network/tc_net_poll.c (hunk 2 above)
Comment (deepaksrma) [reply apps/examples/testcase/le_tc/network/tc_net_poll.c]: both are same.
`#define TC_ASSERT_GEQ(api_name, var, ref) TC_ASSERT_GEQ_CLEANUP(api_name, var, ref, ((void)0)`

File: os/drivers/os_api_test/network/test_net_poll.c (hunk 6 above)
Comment (deepaksrma) [reply os/drivers/os_api_test/network/test_net_poll.c]: this is self explanatory, i think no need

File: apps/examples/testcase/le_tc/network/tc_net_poll.c (hunk 1 above)
Comment (deepaksrma) [reply apps/examples/testcase/le_tc/network/tc_net_poll.c:32]: corrected the order and we did n't need of tinyara/config.h file here.
so didn't include.

File: os/drivers/os_api_test/network/test_net_poll.c [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static int test_poll(unsigned long arg)
+{
+	int ret = -1;
+	poll_test_args_t *args = (poll_test_args_t *)arg;
+	if (args == NULL) {
+		printf("args is NULL\n");
```
Comment (kishore-sn) [inline os/drivers/os_api_test/network/test_net_poll.c]: Please use dbg instead of printf.

Comment (deepaksrma) [reply os/drivers/os_api_test/network/test_net_poll.c]: done

**Pull request closed without merging**